crates/wasi-provider/src/wasi_runtime.rs

```diff
@@ -168,12 +168,12 @@ impl WasiRuntime {
                 .envs(&data.env)
                 .stdout(wasi_common::OsFile::try_from(output_write.try_clone()?)?)
                 .stderr(wasi_common::OsFile::try_from(output_write.try_clone()?)?);
-            let mut ctx_builder_unstable = wasi_common::old::snapshot_0::WasiCtxBuilder::new();
+            let mut ctx_builder_unstable = wasi_common::WasiCtxBuilder::new();
```

Comment:
I'm not very familiar with WASI in wasmtime. I guess `old__snapshot_0` is the first experimental implementation of WASI. The change to use `wasi_common::WasiCtxBuilder` is taken from the PR removing `old::snapshot_0`: https://github.com/bytecodealliance/wasmtime/pull/2444/files#diff-2218893006e42ddf80cedd8282a70b44af7fbb2b1423527dfcb1666a895ed211L366

Reply:
Yeah, `old_snapshot_0` was explicitly referencing the [snapshot phase](https://github.com/WebAssembly/WASI/tree/main/phases) of WASI.